README.md

```diff
@@ -694,6 +695,46 @@ setTrackingProperty: (system: 'palitagem' | 'medallia', name: string, value?: st
 -   `name`: name of the property
 -   `value`: value of the property (nullable)
 
+### setActionBehavior
+
+Method that allows defining an specific behavior (such as showing a
+confirmation) before the specific native actions are executed. This method also
+allows disabling any previous behaviors set.
+
+-   Available for app versions 12.7 and higher
+
+```typescript
+type ActionBehavior = {
+    behavior: 'confirm' | 'default' | 'cancel';
+    title?: string;
+    message?: string;
+    acceptText?: string;
+    cancelText?: string;
+};
+
+setActionBehavior: (actions: {close?: ActionBehavior, back?: ActionBehavior}) => Promise<void>;
+```
+
+`close` and `back` actions are currently available:
+
+-   `back`: Action bar back button pressed (Also for physical back button in
+    android)
+-   `close`: Action bar close button pressed. Includes both “X” and “Close”
+    buttons.
+
+Both have same allowed json parameters, and 3 allowed behaviors:
+
+-   `confirm` Show a confirmation dialog with the required title and message. If
+    no accept or cancel texts are specified, native default ones will be used.
```

Comment:
Just changed this description before reading you were already implementing it 😄 No defaults will be used actually.